xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -342,7 +342,11 @@ bool CGUIControlFactory::GetTexture(const TiXmlNode* pRootNode, const char* strT
   if (!pNode) return false;
   const char *border = pNode->Attribute("border");
   if (border)
+  {
     GetRectFromString(border, image.border);
+    const char *borderinfill = pNode->Attribute("infill");
```

Comment:
pretty sure you'll get a clang format warning on this

`const char* borderinfill`

Reply:
Roger.

Just saying, the whole file is littered with it :).

Reply:
https://github.com/barisione/clang-format-hooks

Reply:
Done.